index.js

```diff
@@ -112,19 +112,36 @@ export const createQueryParameters = ({
 };
 
 export default function open(params) {
-	createOpenLink(params)();
+	createOpenLink(params) //()
 }
```

Comment:
Due to the await call to Linking.canOpenURL(...), I had to make `createOpenLink` an async function and return an await. This creates the delayed effect you had made in the first place. I hope this is OK?